x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/score/DecayTests.java

```diff
@@ -102,6 +102,33 @@ public static Iterable<Object[]> parameters() {
         // Long random
         testCaseSuppliers.addAll(longRandomTestCases());
 
+        // Unsigned Long Linear
+        testCaseSuppliers.addAll(unsignedLongTestCase(0L, 10L, 10000000L, 200L, 0.33, "linear", 1.0));
```

Comment:
Check how [unsigned long test cases](https://github.com/elastic/elasticsearch/blob/f7dd6049d5e3d72a28f57590aff33765f5c3a273/x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/TestCaseSupplier.java#L959) are generated for some ideas on how to generate numbers that don't fit into long